tests/jobs/test_scheduler_job.py

```diff
@@ -111,24 +112,32 @@ def clean_db():
         # The tests expect DAGs to be fully loaded here via setUpClass method below
 
     @pytest.fixture(autouse=True)
-    def set_instance_attrs(self, dagbag):
-        self.dagbag = dagbag
+    def per_test(self) -> Generator:
         self.clean_db()
         self.scheduler_job = None
+
```

Comment:
Here I noticed that the fixtures were a) partially duplicated (clean_db and scheduler_job cleanup was repeated in the "set_instance_attrs" fixture for some reason) and also they were a bit "assymetrical" - When I do fixtures like this I try to make sure what  is usually introduced before yield is "cleaned-up" in reverse orther after yield. Now both fixtures have no repetitive code and they exhibit this "symmetry"

Reply:
It was fixed in the meanitme I think but I think keeping the two fixtures makes sens still.